Generals/Code/Libraries/Source/WWVegas/WW3D2/texture.cpp

```diff
@@ -470,13 +450,9 @@ unsigned int TextureClass::Set_Priority(unsigned int priority)
 
 // ----------------------------------------------------------------------------
 
-void TextureClass::Set_Mip_Mapping(FilterType mipmap)
+void TextureClass::Set_Mip_Mapping(TextureFilterClass::FilterType mipmap)
 {
-	if (mipmap != FILTER_TYPE_NONE && Get_Mip_Level_Count() <= 1) {
-		WWASSERT_PRINT(0, "Trying to enable MipMapping on texture w/o Mip levels!\n");
-		return;
```

Comment:
This test and return is now gone (it is commented inside `Set_Mip_Mapping`). Is that ok?

Reply:
tomsons26
> it would go away with full port of ZHs code anyway, that said I guess I can restore it for now in TextureClass tho probably my next PR would had been to replace the wrappers so kind of mute..